examples/nextjs-auth/package.json

```diff
@@ -1,17 +1,17 @@
 {
-  "name": "with-supabase-auth",
-  "version": "1.0.0",
-  "scripts": {
-    "dev": "next dev",
-    "build": "next build",
-    "start": "next start"
-  },
-  "dependencies": {
-    "@supabase/supabase-js": "^1.2.1",
-    "next": "latest",
-    "react": "^16.13.1",
-    "react-dom": "^16.13.1",
-    "swr": "0.2.3"
-  },
-  "license": "MIT"
-}
+    "name": "with-supabase-auth",
```

Comment:
It looks like you've used a code formatter which adds 4 spaces (or tabs)? We use this prettier config:
https://github.com/supabase/supabase/blob/master/.prettierrc

Is this file formatted with the correct config?